implementations/rust/ockam/ockam_identity/src/secure_channels/secure_channels.rs

```diff
@@ -122,21 +122,8 @@ impl SecureChannels {
     /// Stop a SecureChannel given an encryptor address
     pub async fn stop_secure_channel(&self, ctx: &Context, channel: &Address) -> Result<()> {
         if let Some(entry) = self.secure_channel_registry.unregister_channel(channel) {
```

Comment:
We now do it inside the Decryptor shutdown, right? So we don't need to do it here. `stop_secure_channel` function should be one line

Reply:
So it should look like this:
```rust
    /// Stop a SecureChannel given an encryptor address
    pub async fn stop_secure_channel(&self, _ctx: &Context, channel: &Address) -> Result<()> {
        self.secure_channel_registry.unregister_channel(channel);
        Ok(())
    }
```

@SanjoDeundiak

Reply:
or it should just return `Ok(()))` ?

Reply:
Yes, look correct